app/models/landing_page/block/document_list.rb

```diff
@@ -12,6 +12,7 @@ def initialize(block_hash, landing_page)
       super
 
       @taxon_base_path = data["taxon_base_path"]
+      @heading = data["heading"]
```

Comment:
You could set the default value here, and save a line of ERB in the view:

```ruby
      @heading = data.fetch("heading", "Latest Updates")
```

(Also makes it easier to write a test for the default, if you're into that)

Reply:
Really, these defaults should be in the locales.